config/toml.go

```diff
@@ -284,6 +284,11 @@ external_address = "{{ .P2P.ExternalAddress }}"
 # Comma separated list of seed nodes to connect to
 seeds = "{{ .P2P.Seeds }}"
 
+# Comma separated list of peers to be added to the peer store
+# on startup. Either BootstrapPeers or PersistentPeers are
+# needed for peer discovery
```

Comment:
Maybe write down what the difference is between them although most people already know.

Reply:
The affirmation is not even true. One node can discovery peers by only having `SeedNodes` config parameter set.

Reply:
So, here there are several points which are not clear:

1. A node can discovery peers provided it has any address on its address book (`BootrstapPeers` flag ensures this), any peer configured as persistent, or any address configured as seed node
2. Persistent peers is not meant for peer discovery. It is mostly used by validator nodes that don't run the PEX reactor, namely, don't participate on the peer discovery protocol
3. There is no peer store. There is an address book with addresses of _potential_ peers